apps/publish/content/common.py

```diff
@@ -576,7 +576,9 @@ def _publish_associated_items(self, original, updates={}):
 
         associations = original.get(ASSOCIATIONS) or {}
         for associations_key, associated_item in associations.items():
-            if type(associated_item) == dict and associated_item.get(config.ID_FIELD):
+            if (type(associated_item) == dict
+                    and associated_item.get(config.ID_FIELD)
+                    and associated_item.get('_fetchable')):
```

Comment:
imo would be good to use some default here so it won't change behaviour when `_fetchable` is missing:
```py
and associated_item.get('_fetchable', True) != False
```